### PR TITLE
Elle/further tweaks and improvements

### PR DESCRIPTION
**Pull request by elleneub**

### CODE REVIEW

File: UI/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "style-loader": "^1.1.3"
     },
     "scripts": {
-        "start": "npm install && react-scripts start",
+        "start": "react-scripts start",
```
Comment (elleneub) [inline UI/package.json:26]: This was kinda bugging me, cause sometimes I didn't need to reinstall when restarting. How's that feel to you Megan?

File: ui/jsconfig.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "baseUrl": "src",
+        "paths": {
+            "@components/*": ["src/components/*"]
+        }
+    }
+}
```
Comment (elleneub) [inline ui/jsconfig.json:8]: This make it so we can use absolute paths to the components directory, and just cleans things up

File: UI/package.json (hunk 1 above)
Comment (solgamd) [reply UI/package.json:26]: Agreed

File: ui/jsconfig.json (hunk 2 above)
Comment (solgamd) [reply ui/jsconfig.json:8]: Sounds good to me!

**Review by solgamd — APPROVED** [review]

**Pull request merged**